dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -202,6 +203,12 @@ private LinkageChecker(
             new SymbolProblem(symbol, ErrorType.INCOMPATIBLE_CLASS_CHANGE, containingClassFile));
       }
 
+      // Check the existence of the parent class or interface for the class
+      Optional<SymbolProblem> parentSymbolProblem = findParentSymbolProblem(targetClassName);
+      if (parentSymbolProblem.isPresent()) {
+        return parentSymbolProblem;
+      }
+
```

Comment:
The problem was that, if any of superclasses or interfaces was missing, the subsequent operations to check method existence in them would fail. Screenshot of the stacktrace in #933